### ISSUE TITLE
Allow for .md file format as an alternative to .txt

### ISSUE THREAD

**Issue by hamishlow** [vkbo/novelWriter#1960]
One suggestion which would be really really useful for me, and would extend the usability of novelWriter, would be if we could use .md files instead of .txt. In this way I could have the novelWriter project within my Obsidian vault, and use Obsidian for notes, and novelWriter for structure... two tools for writing. novelWriter is far superior to what is currently available in Obsidian (longform plugin)... 

Otherwise I can copy-paste my notes into novelWriter, or convert to txt - which makes them useless in obsidian.  If novelWriter used .md it would make both tools useful and valid with the same set of notes.

**Comment by vkbo** [comment]
There are no `.txt` files in novelWriter. Do you mean the files in the `content` folder?

I would strongly advice against mixing a novelWriter project into an Obsidian project. Firstly, novelWriter isn't pure markdown, and contains a mix of plain text formatting styles plus its own metadata syntax. I'm not sure how Obsidian would interact with that. In addition, novelWriter does not expect its project files to be edited outside of the app. It has some logic to handle errors and unexpected changes, but those are only ment as a safeguard.

**Comment by hamishlow** [comment]
Thanks for your reply Veronica,

Yes I meant the files in the content folder.

I guess I'll have to copy and paste content into novelWriter then as there's nothing else comparable out there. Most of my notes are in .md.

Thanks for your great work!

Kind regards,

Hamish

**Comment by vkbo** [comment]
Yeah, it just isn't designed for this, so actively supporting this would come with a host of problems. Also, the file names aren't suitable for this usage. They are obscured for a reason, to discourage tampering and to not have to deal with file name changes and conflicts in the project.

A better solution here would probably be some other integration with Obsidian. This may be worth looking into in another way. Let's keep the ticket open, and I'll re-label it.

**Comment by hamishlow** [comment]
Hi Veronica, I've switched from using Obsidian to Logseq, but still keen for someway to pull content into novelwriter easily when I get past an initial draft...

**Comment by vkbo** [comment]
It is still not clear to me what sort of interaction would be needed. Import should already work fine, so I understood this to be more about cross-linking in some way or another.
